server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionService.java

```diff
@@ -952,4 +955,16 @@ private void maskConnectionHost(ConnectionConfig connection) {
         }
     }
 
+    private Date getEarliestObjectSyncTime(@NotNull Long connectionId) {
```

Comment:
dup code?

Reply:
Not duplicated, the Project and DataSource has different codes.